src/Compilers/Core/Portable/SpecialType.cs

```diff
@@ -263,9 +263,14 @@ public enum SpecialType : sbyte
         /// </summary>
         System_Runtime_CompilerServices_InlineArrayAttribute = 46,
 
+        /// <summary>
+        /// Indicates that the type is <see cref="ReadOnlySpan{T}"/>
```

Comment:
>ReadOnlySpan [](http://example.com/codeflow?start=50&length=12)

Should probably use fully qualified name here to indicate that this is just a type with the given name, not some "blessed" type.

Reply:
Using xml documentation tags makes it very easy to read this with proper IDE support. Moreover, it enables navigating to the definition of the type just by clicking the link in the hover:
![devenv_CWPMw6NXaU](https://github.com/dotnet/roslyn/assets/70431552/cafce35d-8987-4ba0-9d89-52c3057683c3)
I believe if we just specify qualified name there like `System.ReadOnlySpan<T>` we would loose more than we gain. If I misunderstood you and you have some other idea in mind please use inline suggestion feature of GH review, this is quite easy to do just from the browser

Reply:
I am symply suggesting to add "System."

Reply:
I meant to add "System." into the cref

Reply:
It would be embarrassing If I somehow messed this one up again